src/cpu/pred/tage.cc

```diff
@@ -62,7 +62,9 @@ namespace gem5
 namespace branch_prediction
 {
 
-TAGE::TAGE(const TAGEParams &params) : BPredUnit(params), tage(params.tage)
+TAGE::TAGE(const TAGEParams &params) :
```

Comment:
```suggestion
TAGE::TAGE(const TAGEParams &params)
      :  ConditionalPredictor(params),
         tage(params.tage)
```
Make this also consistent with the conditionalbranch predictor constructor